tests/plugins/easyimage/manual/upload.md

```diff
@@ -1,6 +1,7 @@
 @bender-tags: 4.9.0, feature, 932
 @bender-ui: collapsed
 @bender-ckeditor-plugins: sourcearea, wysiwygarea, floatingspace, toolbar, easyimage, undo
+@bender-include: ./_helpers/tools.js
```

Comment:
This file is actually already loaded in [the upload HTML file](https://github.com/ckeditor/ckeditor-dev/blob/b6c3dcc/tests/plugins/easyimage/manual/upload.html#L24).

There are more cases like that, please fix them to avoid redundant file loading.